runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/TestCountingSource.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
`CountingSource` returns a vanilla `UnboundedSource`. We could easily make it return the more concrete type.